docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -151,6 +151,63 @@ to perform the remaining query processing on a smaller amount of data.
 The specifics for the supported pushdown of table joins varies for each data
 source, and therefore for each connector.
 
+However, there are some generic conditions that must be met in order for a join
+to be pushed down:
+
+* join pushdown must be enabled
+* all predicates that are part of the join must be possible to be pushed down
+* the tables in the join must be from the same catalog
+
+You can check if pushdown for a specific join is performed by looking at the
+:doc:`EXPLAIN plan </sql/explain>` of the query. If a join is successfully
+pushed down to the connector, the explain plan does **not** show that ``Join`` operator.
+The explain plan only shows the operations that are performed by Trino.
+
+As an example, we loaded the TPCH data set into a PostgreSQL database and then
+queried it using the PostgreSQL connector::
+
+    SELECT c.custkey, o.orderkey
+    FROM orders o JOIN customer c ON c.custkey = o.custkey;
+
+You can get the explain plan by prepending the above query with ``EXPLAIN``::
+
+    EXPLAIN SELECT c.custkey, o.orderkey
+    FROM orders o JOIN customer c ON c.custkey = o.custkey;
+
+The explain plan for this query does not show any ``Join`` operator,
```

Comment:
The following plan results from the PostgreSQL connector querying TPCH data in a PostgreSQL database. It does not show any ``Join`` operator as a result of the successful join push down::